staging/src/k8s.io/client-go/scale/interfaces.go

```diff
@@ -36,4 +37,8 @@ type ScaleInterface interface {
 
 	// Update updates the scale of the given scalable resource.
 	Update(resource schema.GroupResource, scale *autoscalingapi.Scale) (*autoscalingapi.Scale, error)
+
+	// Patch patches the scale of the given scalable resource.
+	// Only Merge patch and JSON patch could be used here.
```

Comment:
> Only Merge patch and JSON patch could be used here.

It's up to the server what patch types it accepts... nothing client-side requires this to be mergepatch/jsonpatch only

Reply:
hmmm but now the server did only support these two types of patch: https://github.com/kubernetes/kubernetes/blob/1871f75b32f5d96c62ce6aa8c2d393124b7002aa/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go#L269-L272

I think it may be better to note it.

Reply:
That's for CRDs specifically, not all API endpoints

Reply:
oh sorry... got it!

Reply:
fixed